lib/browserDocument.js

```diff
@@ -98,15 +98,7 @@ Document.$emitter = new EventEmitter();
  */
 
 Document.prototype._execDocumentPreHooks = function _execDocumentPreHooks(opName) {
```

Comment:
```suggestion
Document.prototype._execDocumentPreHooks = async function _execDocumentPreHooks(opName) {
```

To be in style with the other changes, this should likely also be marked async